test/apps/errors/src/routes/preload-reject.svelte

```diff
@@ -0,0 +1,7 @@
+<script context="module">
+	export function preload() {
+    return Promise.reject(new Error('boom'))
```

Comment:
Convert spaces to tabs

```suggestion
		return Promise.reject(new Error('boom'))
```